### PR TITLE
[Refact] - Call helper method instead redefining ms/frame

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: libse/Paragraph.cs [hunk 1]
```diff
@@ -114,12 +114,8 @@ public void Adjust(double factor, double adjustmentInSeconds)
         {
             if (StartTime.IsMaxTime)
                 return;
-
-            double seconds = StartTime.TimeSpan.TotalSeconds * factor + adjustmentInSeconds;
-            StartTime.TimeSpan = TimeSpan.FromSeconds(seconds);
-
-            seconds = EndTime.TimeSpan.TotalSeconds * factor + adjustmentInSeconds;
-            EndTime.TimeSpan = TimeSpan.FromSeconds(seconds);
+            StartTime.TotalSeconds *= factor + adjustmentInSeconds;
```
Comment (niksedk) [inline libse/Paragraph.cs]: Gives a wrong result... I've added some unit tests :)

Comment (ivandrofly) [inline libse/Paragraph.cs]: > Gives a wrong result... I've added some unit tests :)

Hmmm interesting, any idea what causes the wrong result?

Comment (niksedk) [inline libse/Paragraph.cs]: *= just multiplies, so it gives StartTime.TotalSeconds = StartTime.TotalSeconds * (actor + adjustmentInSeconds)
Did you test it?

Comment (ivandrofly) [inline libse/Paragraph.cs]: > *= just multiplies, so it gives StartTime.TotalSeconds = StartTime.TotalSeconds * (actor + adjustmentInSeconds)
> Did you test it?

Aaah okay, So *= will distribute `StartTime.TotalSeconds` :+1:

**Pull request closed without merging**